docs/firstobject.md

```diff
@@ -76,78 +76,81 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 
-As the name of this step suggests; on this page you will be adding all your Linked Data elements. This Step showcases some of the Autocomplete Linked Data Webform elements we built for Archipelago. We truly believe in Wikidata as an open, honest, source of Linked Open Data and also one where you can contribute back. But we also have LoC autocompletes and Getty.
+Again, enter all fields with a red `*` and when you are finished, click `Move on to next step`
 
-Click: `Next: Collections`
+![Ingest Step 2 - Linked Data](../imgs/02_ingest-step-two_linked-data.png)
 
 ### Collections
 
-![Ingest Step3](../imgs/inges-step-3_Collections.jpg)
-
 Since this is our first digital object we do not yet have a Digital Object *Collection* for which `My First Digital Object` could be a member of. In other words, you can leave `Collection Membership` blank and click `Next: Upload Files`.
 
+![Ingest Step 3 - Collections](../imgs/03_ingest-step-three_collections.png)
+
 ### Upload Files
 
-![Ingest Step4](../imgs/ingest-step-4_Upload-files.jpg)
+Now we will upload our `Photograph`. Click `Choose Files` to open your file selector window and choose which file you would like to ingest.
 
-This step is pretty straight forward: Click `Choose Files` to open your file selector window and choose which file you would like to ingest. After you've done this, click `Save Metadata`. 
+![Ingest Step 4 - Attach Files pt 1](../imgs/04_ingest-step-four_upload-files_1.png)
 
-Two things to note: The `Save Metadata` button simply persists the metadata in the current webform session. The actual ingest of the Object happens when you click `Save` (see the **Complete** step below). Secondly, each "upload field" has a set of allowed file types (such as .jpg, .tiff, .png) and in this demo there are also file size restrictions which are explained in the Upload Field's description.
+Once you've uploaded your file, you will see all the Exif data extracted from the image.
 
-From here, Steps 5 & 6 (hotspots & Preview) are skipped and you are directed to the **Complete** step.
+![Ingest Step 4 - Attach Files pt 2](../imgs/04_ingest-step-four_upload-files_2.png)
 
-### Complete
+Once you've mentally digested all of that data, let's go ahead and click `Save Metadata`.
+
+**Question:** "Wait! Don't I need to switch `Save as` from *Draft* to *Publish*? I'm ready for this Digital Object to be shared with the world!"
 
-![Ingest Step7](../imgs/ingest-step-7_Complete.jpg)
+**Answer:** You can, but it's not necessary to switch to *Publish* now. By clicking `Save Metadata` we are simply persisting the metadata in the current webform session. So, we can confidently leave the status as *Draft* as we click `Save Metadata` because all we're doing here is... well, *saving* all the *metadata* we've entered so far.
```

Comment:
Without `*Publish*` they won't see the object appear in the search. So let's encourage them to make it published.